### ISSUE TITLE
Support new OpenAI models with larger context

### ISSUE THREAD

**Issue by humphd** [tarasglek/chatcraft.org#106]
We should update our list of models we can choose from, so people with access to the newer large-context models can access them.  Simon wrote about it: https://til.simonwillison.net/llms/larger-context-openai-models-llm

**Comment by tarasglek** [comment]
Found an api endpoint to list em
```
curl https://api.openai.com/v1/models   -H "Authorization: Bearer $OPENAI_API_KEY" |jq .data| jq .[].id|grep gpt
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 37711  100 37711    0     0  62273      0 --:--:-- --:--:-- --:--:-- 62747
"gpt-3.5-turbo-16k-0613"
"gpt-4-0613"
"gpt-4"
"gpt-4-0314"
"gpt-3.5-turbo"
"gpt-3.5-turbo-0301"
"gpt-3.5-turbo-16k"
"gpt-3.5-turbo-0613"
```

**Comment by humphd** [comment]
Fixed by #107
